server/controllers/AuthController.js

```diff
@@ -2,13 +2,17 @@ import bcrypt from 'bcrypt';
 import dotenv from 'dotenv';
 import jwt from 'jsonwebtoken';
 import { Op } from 'sequelize';
+import crypto from 'crypto';
 import db from '../database/models';
 import Users from '../utils/utilities';
+import sendVerifyEmailMessage from './helpers/emailSender';
+
 
 dotenv.config();
 
 const { User } = db;
 const secret = process.env.SECRET_KEY;
+const secret2 = process.env.SECRET_KEY2;
```

Comment:
use a more descriptive name for SECRET_KEY2

Reply:
I have changed it to EMAIL_SECRET_KEY.